src/main/java/com/stripe/param/EphemeralKeyCreateParams.java

```diff
@@ -0,0 +1,81 @@
+package com.stripe.param;
```

Comment:
There's no change to this file from the generator. But I removed the comment that is generated.
```
 // Generated by com.stripe.generator.entity.SdkBuilder
```